setup.py

```diff
@@ -38,6 +38,7 @@ def readme():
     extras_require={
         "testing": ["basx-bread[testing]"],
         "mailer-integration": ["mailchimp_marketing"],
+        "pycountry": ["mailchimp_marketing"],
```

Comment:
I think this should be:

    "mailer-integration": ["mailchimp_marketing", "pycountry",

But more importantly: Could you use the functionality from the package django-countries? It is a default dependency of bread because a country data type is common requirement.

Bwt. is MMERGE11 a "standard" mailchimp field or urban mosaic specific?

Reply:
oh yes, that's a typo :)

I tried to use the django-countries, but I did not find the functionality which I needed... (getting from an English country name to the alpha_2 code).
If you can give me a hint on how to make it work I will use django-countries.

Reply:
I don't think "MMERGE11" is standard. Sadly it seems that urbanmosaic is not using the standard format for addresses. Probably they started using addresses before there was even a standard way in Mailchimp. I think as soon as its "MMERGE<someNumber>" it's a custom field.
I will change it to use the "standard" format by default and override it in the urban mosaic settings.